packages/f-checkout/src/components/Selector.vue

```diff
@@ -1,42 +1,21 @@
 <template>
-    <div
+    <form-field
+        id="time-selection"
         data-test-id="form-select"
-        :class="[
-            $style['o-form-select'],
-            (selectedTime ? $style['o-form-select--float'] : '')
-        ]">
-        <label
-            for="time-selection"
-            data-test-id="fulfillment-time-label"
-            :class="$style['o-form-select-label']">
-            {{ orderMethod }}
-        </label>
-        <select
-            id="time-selection"
-            v-model="selectedTime"
-            :class="$style['o-form-select-input']"
-            data-test-id="fulfillment-time"
-            @change="selectionChanged">
-            <option
-                v-for="(time, index) in fulfillment.times"
-                :key="index"
-                :value="time.from">
-                {{ time.label.text }}
-            </option>
-        </select>
-    </div>
+        :class="$style['c-selector']"
+        input-type="dropdown"
+        :label-text="orderMethod"
+        :dropdown-options="fulfillmentTimes"
+        @input="selectionChanged" />
 </template>
 
 <script>
 import { mapState } from 'vuex';
+import FormField from '@justeat/f-form-field';
```

Comment:
I think it's worth calling this the form element it represents so at glance it's easier to work out the type of element we're using?